tests/test_tensorflow_keras_models.py

```diff
@@ -410,11 +410,17 @@ def test_file_properties(tmpdir):
     tiledb_obj = TensorflowTileDB(uri=tiledb_array)
     tiledb_obj.save(model=model)
 
-    assert tiledb_obj._file_properties["ML_FRAMEWORK"] == "TENSORFLOW"
-    assert tiledb_obj._file_properties["STAGE"] == "STAGING"
-    assert tiledb_obj._file_properties["PYTHON_VERSION"] == platform.python_version()
-    assert tiledb_obj._file_properties["ML_FRAMEWORK_VERSION"] == tf.__version__
-    assert tiledb_obj._file_properties["PREVIEW"] == ""
+    assert tiledb_obj._file_properties["TILEDB_ML_MODEL_ML_FRAMEWORK"] == "TENSORFLOW"
```

Comment:
This should be TENSORFLOW KERAS, but I guess you will have to merge #63 first and then rebase :)